content/en/docs/guides/packaging-an-application.md

```diff
@@ -115,62 +116,110 @@ metadata:
 
 ### Build & Validate
 
-If you just want to validate the local package contents, use:
+If you just want to validate the local package contents, run the
+`kubectl package validate` command:
 
 ```sh
-$ kubectl package validate 1_applications/1_start
+kubectl package validate 1_applications/1_start
+```
+
+\
+For example, if you encounter an issue such as a missing annotation, the output might look like this:
 
-# example:
+```sh
 Error: Package validation errors:
 - Missing package-operator.run/phase Annotation in deployment.yaml#0
 ```
 
 ---
-To inspect the parsed hierarchy of your package, use:
+To inspect the parsed hierarchy of your package, run the `kubectl package tree` command:
 
 ```sh
-$ kubectl package tree 1_applications/1_start
+kubectl package tree 1_applications/1_start
+```
 
-# example:
+\
+Example output:
+
+```sh
 nginx
 Package namespace/name
 └── Phase deploy
     └── apps/v1, Kind=Deployment /nginx-deployment
 ```
 
 ---
-And finally to build your package as a container image use:
+Finally, to build your package as a container image use one of the following methods:
 
-```sh
-# -o will directly output a `podman/docker load` compatible tar.gz
-# of your container image.
-# Use this flag if you don't want to push images to a container registry.
-# To give a demonstration, you can use nginx:local as your image url here.
-$ kubectl package build -t <your-image-url-goes-here> -o nginx.tar.gz 1_applications/1_start
-# example: load image into kind nodes:
-$ kind load image-archive nginx.tar.gz
-
-# Alternatively push images directly to a registry.
-# Assumes you are already logged into a registry via `podman/docker login`
-$ kubectl package build -t <your-image-url-goes-here> --push 1_applications/1_start
-```
+**Method 1:**
+
+1. Utilize the `-o` flag to directly generate a tar.gz file
+compatible with podman/docker load.
+This option is suitable if you do not intend to push images to a container registry:
+
+    ```sh
+    kubectl package build -t <your-image-url-goes-here> -o <output-file-name>.tar.gz <path-to-package-contents>
+    ```
+
+    \
+    Example:
+
+    ```sh
+    kubectl package build -t nginx:local -o nginx.tar.gz 1_applications/1_start
+    ```
+
+\
+2. Load the image into kind or minikube nodes.
+
+* For kind nodes use:
+
+    ```sh
+    kind load image-archive nginx.tar.gz
+    ```
+
+* For minikube nodes, use:
 
-{{< alert text=`If you load the image to a local registry (e.g. with
-'minikube image load' or 'kind load'), specify a tag different than 'latest'
-(e.g. 'test' or 'v1'), otherwise Package Operator won't be able to load it.` />}}
+    ```sh
+    minikube image load nginx.tar.gz
+    ```
+
+    {{< alert text=`When loading the image to a local registry
+    using 'minikube image load' or 'kind load', specify a tag
+    other than 'latest', otherwise Package Operator will not be
+    able to load it.` />}}
+
```

Comment:
I think there is still a step missing here. When I load the image in minikube, I can see it listed when I do a:
minikube image ls
docker.io/library/nginx:0.0.1

But when I try to deploy it using the local image, I always get an error of: 
```
Message:               GET https://index.docker.io/v2/library/nginx/manifests/0.0.1: MANIFEST_UNKNOWN: manifest unknown; unknown tag=0.0.1
    Observed Generation:   1
    Reason:                ImagePullBackOff

```

I have the image url in the package.yml set to:
spec:
  # Image specifies when the package that shall be installed
  # is found at.
  image: "nginx:0.0.1"

Reply:
I believe this step is building your package as a container image and then loading the local image into minikube. The part that isn't working then, is pulling the local image that was loaded. 

Note - if I build the image and push it to quay and set the image to quay url, this step works fine.

Reply:
removed option to load archive as all images must be pulled from a repository now.